src/js/util/game.js

```diff
@@ -1,6 +1,20 @@
+import { WINNER_CONGURATULATION_DELAY } from './constant.js';
+import { GAME_MESSAGE } from './message.js';
+
 export const getRandomNumber = ({ max, min }) => {
   return Math.floor(Math.random() * (max - min + 1)) + min;
 };
 
 export const modifyCarNameInputValue = value =>
   value.split(',').map(name => name.trim());
+
+export const delay = ms => {
+  return new Promise(res => {
+    setTimeout(res, ms);
+  });
+};
```

Comment:
```suggestion
export const delay = ms => new Promise(resolve => setTimeout(resolve, ms))
```

바로위 modifyCarNameInputValue에서는 expression을 바로 return할때 bracket과 return문을 잘 생략하여 작성하셨는데 여기는 길게 쓰셨네요 ㅎㅎ

res는 response에도 많이 쓰이는 줄임말이라 resolve라고 길게 써주시는게 좋습니다.

Reply:
bdef161

그렇군요! 앞으로 `resolve, reject`로 길게 작성하겠습니다 :)